wiki/Beatmap/Pattern/Mania/Streams/en.md

```diff
@@ -0,0 +1,41 @@
+---
+stub: true
+tags:
+  - streams
+---
+
+# Streams
+
+_For specific stream types unrelated to 4-key, refer to [chordstreams](mania/chordstreams)._
+
+**Streams** are constant notes within the same snap interval that occur for at least one beat, avoiding an occurance of a [minijack](/wiki/Beatmap/Pattern/Mania/Jacks/en.md#Minijack). There are many different types of streams that impose different levels of strain on the player. When stream is used on its own, it's likely referring to a **single note stream**.
+
+![](img/stream.png "This image shows a specific type of stream known as a roll.")
+
+A **roll** is a common variation of a stream consisting of notes that go from the first to the last column or last to the first column.
+
+**Streams** that appear shorter than one beat are typically referred to as **bursts**, and are more of a shorter quick succession of notes. They commonly use snaps higher than 1/4th.
+
+![](img/burst.png)
+
+## Jumpstream
+
+**Jumpstreams** are streams that involve jumps. Usually, the stream itself is on 1/4th snaps whilst the jumps land on snaps greater than 1/4th. This is the most common type of stream pattern used in 4-key osu!mania.
+
+![](img/jumpstream.png)
+
+ It's also loosely referred to as a type of chordstream, which mixes different types of chords and streams.
+
+## Handstream
+
+**Handstreams** are streams that involve hands, and may involve jumps as well. Similar to jumpstreams, the stream itself is on 1/4th snaps whilst the chords can land on snaps greater than 1/4th.
+
+![](img/handstream.png)
```

Comment:
Would use an image that features both hands and jumps here for clearer explanation.

Reply:
Mehhh, I think being communicative that it involves hands and is not limited to hands is enough to specify that jumps can occur. I just think that an image with jumps in it as well would come off more confusing than anything for people exposing themselves to the term for the first time.